data/monster/undeads/undead_elite_gladiator.lua

```diff
@@ -73,24 +73,24 @@ monster.voices = {
 }
 
 monster.loot = {
-	{name = "platinum coin", chance = 44000, maxCount = 40},
-	{name = "belted cape", chance = 500},
-	{name = "ultimate health potion", chance = 500, maxCount = 2},
-	{name = "throwing star", chance = 500, maxCount = 18},
-	{name = "knight axe", chance = 250},
-	{id = 3307, chance = 500}, -- scimitar
-	{name = "protection amulet", chance = 250},
-	{name = "hunting spear", chance = 250},
-	{name = "broken gladiator shield", chance = 250},
-	{name = "dark helmet", chance = 250},
-	{name = "flask of warrior's sweat", chance = 250},
-	{name = "plate armor", chance = 250},
-	{name = "plate legs", chance = 250},
-	{name = "two handed sword", chance = 250},
-	{name = "great health potion", chance = 250, maxCount = 2},
-	{id = 3049, chance = 230}, -- stealth ring
-	{name = "crusader helmet", chance = 150},
-	{name = "relic sword", chance = 150}
+	{name = "platinum coin", chance = 100000, maxCount = 40},
+	{name = "belted cape", chance = 2448},
```

Comment:
isn't that ~2,4%?

Reply:
[Belted Cape](https://tibia.fandom.com/wiki/Belted_Cape) (24.48%)

Wiki reports Belted Cape as a drop chance of 24.48%

Reply:
yes, but your change will gives you probably ~2.4% drop chance.

Reply:
Yes... I just realised nearly all my percentages are off by a factor of ten. I will need to redo and resubmit this. Apologies.

Reply:
Is it now up to me to remove this file from the pull request?

Reply:
just fix this in this pull request as its should be and it will be all good.